app/townsquare/migrations/0020_auto_20200521_1020.py

```diff
@@ -12,7 +12,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AlterField(
-            model_name='pinnedpost',
+            model_name='PinnedPost',
```

Comment:
Whats up with this change?

Reply:
As I made the PinnedPost model, in lower case I was giving conflicts in my stay even deleting the entire project in Docker, if it is not necessary for production I can delete this file from the commit.

Reply:
that is very strange - yeah lets remove it and then if we need it we can do a separate PR so we can isolate the changes